demo-videos/README.md

```diff
@@ -0,0 +1,42 @@
+
+# Demo video for each demo activity
+
+##### Demo for application 'Custom Landing Strip'
+
+![Demo for application 'Custom Landing Strip'](/demo-videos/CustomLandingStrip.gif?raw=true)
+
+
+##### Demo for application 'Custom Tab Activity'
+
+![Demo for application 'CustomTabActivity](/demo-videos/CustomTabActivity.gif?raw=true)
+
+
+##### Demo for application 'Fixed With Tab Activity'
+
+![Demo for application 'Fixed With Tab Activity'](/demo-videos/FixedWithTabActivity.gif?raw=true)
+
+
+##### Demo for application 'Multiple Listeners Activity'
+
+![Demo for application 'Multiple Listeners Activity'](/demo-videos/MultipleListenersActivity.gif?raw=true)
+
+
+##### Demo for application 'No Fragments Simple Text Activity'
+
+![Demo for application 'No Fragments Simple Text Activity'](/demo-videos/NoFragmentsSimpleTextActivity.gif?raw=true)
+
+
+##### Demo for application 'Simple Text Tab Activity'
+
+![Demo for application 'Simple Text Tab Activity'](/demo-videos/SimpleTextTabActivity.gif?raw=true)
+
+
+### How to create video
```

Comment:
If you can remove this section, I'll :shipit:

A tutorial on making gifs is a bit out of scope for this repo - we could add something to our base repo and link to it, but it would be more detailed than this. Thanks!